src/main/java/com/contrastsecurity/maven/plugin/ContrastInstallAgentMojo.java

```diff
@@ -387,8 +388,21 @@ Path installJavaAgent(ContrastSDK connection) throws MojoFailureException {
           e);
     }
     // Save the jar to the 'target' directory
-    final Path agent = Paths.get(project.getBuild().getDirectory(), AGENT_NAME);
+    final Path target = Paths.get(project.getBuild().getDirectory());
     try {
+      FileUtils.forceMkdir(target.toFile());
+    } catch (final IOException e) {
+      throw new MojoFailureException("Unable to create directory " + target, e);
+    }
+    final Path agent = target.resolve(AGENT_NAME);
+    try {
+      Path parentDir = agent.getParent().getFileName();
+
+      // If 'target' directory doesn't exist, create it
+      if (!Files.exists(parentDir)) {
```

Comment:
This is redundant: you've already created the directories on line 393. You don't need any of the changes after line 397, right?

Reply:
Agh, the patch got messed up :shaking-angry-fist: